projects/kit/components/tabs/tabs-with-more/tabs-with-more.component.ts

```diff
@@ -216,7 +216,7 @@ export class TuiTabsWithMoreComponent implements AfterViewInit {
         let total =
             tabs.reduce((acc, {scrollWidth}) => acc + scrollWidth, 0) +
             maxIndex * margin -
-            moreWidth;
+            tabs[tabs.length - 1].scrollWidth;
```

Comment:
is it possible NPE?

Reply:
No, this is "more" tab, it is always there.